README.md

```diff
@@ -90,8 +129,26 @@ Place your JavaScript code inside of the provided `client/app` folder. Use modul
   global.HelloWorld = HelloWorldAppServer;
   ```
    
+### Rails View Helpers In-Depth
+Once the bundled files have been generated in your `app/assets/javascripts/generated` folder and you have exposed your components globally, you will want to run your code in your Rails views using the included helper method.
+
+This is how you actually render the React components you exposed to `window` inside of `clientGlobals` (and `global` inside of `serverGlobals` if you are server rendering).
+
+`react_component(component_name, props = {}, options = {})`
++ **react_component_name:** Can be a React component, created using a ES6 class, or `React.createClass`, or a generator function that returns a React component.
++ **props:** Ruby Hash which contains the properties to pass to the react object
++ **options:**
+  + **generator_function:** default is false, set to true if you want to use a generator function rather than a React Component.
+  + **prerender:** enable server-side rendering of component. Set to false when debugging!
+  + **trace:** set to true to print additional debugging information in the browser. Defaults to true for development, off otherwise.
+  + **replay_console:** Default is true. False will disable echoing server-rendering logs to the browser. While this can make troubleshooting server rendering difficult, so long as you have the default configuration of logging_on_server set to true, you'll still see the errors on the server.
++ Any other options are passed to the content tag, including the id
+
+`def server_render_js(js_expression, options = {})`
```

Comment:
@justin808 hmm, the only place that options is called is the very last line, so I think yes on **replay_console** and no on **trace**?

` raw("#{result[0]}#{replay_console(options) ? result[1] : ''}")`

https://github.com/shakacode/react_on_rails/blob/master/app/helpers/react_on_rails_helper.rb#L127